src/main/java/org/openRealmOfStars/starMap/Route.java

```diff
@@ -180,10 +113,9 @@ public boolean isFixing() {
    */
   public void makeNextMove() {
     for (int i = 0; i < ftlSpeed; i++) {
-      if (distance > 0) {
-        startX = startX + mx;
-        startY = startY + my;
-        distance--;
+      if (getDistance() > 0) {
+        startX = startX + getMx();
```

Comment:
Calling getMx() for each moves changes the mx value for each move. Previous route get same internal speed for whole route. I think this might now go wrong.